2021-offseason/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Drivetrain extends SubsystemBase {
+  private CANSparkMax frontLeft, frontRight, backLeft, backRight;
+
+  private DifferentialDrive difDrive;
+  private boolean driveInverted;
```

Comment:
nit: can we try using `final` on all member variables?
```suggestion
  private final CANSparkMax frontLeft, frontRight, backLeft, backRight;

  private final DifferentialDrive difDrive;
  private final boolean driveInverted;
```